pkg/webui/components/key-value-map/entry.js

```diff
@@ -94,6 +95,7 @@ class Entry extends React.Component {
           code
         />
         <Input
+          data-test-id={'value'}
```

Comment:
Why not relying on the `name` attribute which is unique? With this approach each entry will have the same `data-test-id` attribute.